masterlist.yaml

```
@@ -17203,17 +17203,23 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/493/' ]
     group: *earlyLoadersGroup
 
-  - name: 'Hothtrooper44_Armor(Compilation|_Ecksstra).esp'
+  - name: 'Hothtrooper44_Armor(Compilation|_Ecksstra)\.esp'
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/3479/'
         name: 'Immersive Armors'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/85913/'
+        name: 'Immersive Armors Refitted and Reinforced'
   - name: 'Hothtrooper44_ArmorCompilation.esp'
     after: [ 'Ambriel.esp' ]
     msg:
       - <<: *patch3rdParty_LotD
         condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_IA_Patch.esp")'
       - <<: *patch3rdParty_KPH_CCOR
-        condition: 'active("Complete Crafting Overhaul_Remastered.esp") and not active("Hothtrooper44_ArmorCompilation_CCOR_Patch.esp")'
+        condition: 'active("Complete Crafting Overhaul_Remastered.esp") and not active("Hothtrooper44_ArmorCompilation_CCOR_Patch.esp") and not active("IARR, CCOR Patch.esp")'
+      - <<: *requiresX
+        type: error
+        subs: [ '[Immersive Armors - Asdasfa Tweaks and Fixes](https://www.nexusmods.com/skyrimspecialedition/mods/85433/)' ]
+        condition: 'version("Hothtrooper44_ArmorCompilation.esp", "1.2.0IARR", ==) and not active("Immersive Armors - Asdasfa Tweaks and Fixes.esp")'
```

```suggestion
      - <<: *requiresResources
        subs: [ '[Immersive Armors - Asdasfa Tweaks and Fixes](https://www.nexusmods.com/skyrimspecialedition/mods/85433/)' ]
        condition: 'version("Hothtrooper44_ArmorCompilation.esp", "1.2.0IARR", ==) and not file("meshes/armor/ala")'
```
According to the descriptions of both mods, the Asdasfa plugin is contained within the replacer & shouldn't be used with it. It's just the meshes that are required. You could just check if one of the mesh folders is installed.